docs/source/author/topics.rst

```diff
@@ -80,1007 +76,634 @@ the project's topic application, where:
+.. code-block:: yaml
 
-      - For example:
+  unit-plans:
+    - unit-plan
+    - unit-plan-2
 
-        .. code-block:: yaml
+  icon: img/binary-numbers-0-1.png
 
-          number: 1
+  other-resources: other-resources.md
 
-  - Each lesson may also contain any of the following key/value pairs
-    (same indentation as ``minimum-age``, ``maximum-age``, etc):
+  misc-structure-files:
+    - programming-exercises
+    - curriculum-integrations
 
-    - ``programming-exercises`` - A list of slugs for the programming
-      exercises for this lesson.
-      The slugs are defined in the :ref:`programming-exercises-file` for
-      the same topic as the lessons.
-      You can include any number (including zero) of programming exercises,
-      and you can list the same programming exercise for different lessons.
-      The programming exercises listed should be relevant to the content of
-      the lesson, and not the age range of the lesson.
-      This is because programming exercises are written for all ages.
 
-      - For example:
+.. _unit-plan-file:
 
-        .. code-block:: yaml
+Unit Plan Configuration File
+------------------------------------------------------------------------------
 
-          programming-exercises:
-            - count-to-16
-            - count-to-a-million
+- **File Name:** ``<unit-plan-name>.yaml``
 
-    - ``learning-outcomes`` - A list of slugs for the learning outcomes for
-      this lesson.
-      The slugs are defined in the :ref:`learning-outcomes-file`.
+- **Location:** ``topic/content/<language>/<topic-name>/<unit-plan-name>/``
 
-      - For example:
+- **Referenced In:** ``topic/content/<language>/<topic-name>/<topic-name>.yaml``
 
-        .. code-block:: yaml
+- **Purpose:** This file defines all the lessons (and their respective)
+  attributes for the unit plan.
 
-          learning-outcomes:
-            - binary-data-representation
-            - binary-count
-            - binary-convert-decimal
-            - binary-justify-representation
+  - **Required Fields:**
 
-    - ``curriculum-areas`` - A list of slugs for the curriculum areas for
-      this lesson.
-      The slugs are defined in the :ref:`curriculum-areas-file`.
+    - ``<lesson-name>:`` This is the key for the lesson. Each lesson has its own list of
+      required and optional fields:
 
-      - For example:
+      - **Required Fields:**
 
-        .. code-block:: yaml
+        - ``min-age:`` The suggested minimum age group to teach this lesson to.
 
-          curriculum-areas:
-            - maths
+        - ``max-age:`` The suggested maximum age group to teach this lesson to.
 
-    - ``resources-classroom`` - A list of Markdown text of classroom resources
-      required for this lesson resources-classroom.
+        - ``number:`` The number order for this lesson.
+          Lessons are sorted by minimum age, maximum age, then number so lessons in
+          different age ranges can use the same number without conflict.
```

Comment:
Might be better as "Lessons are group by their minimum age and maximum age, then ordered by number so lessons in different age groups can use the same number without conflict."